src/Symfony/Component/Form/Extension/DependencyInjection/DependencyInjectionExtension.php

```diff
@@ -18,11 +18,11 @@
 
 class DependencyInjectionExtension implements FormExtensionInterface
 {
-    private $guesser;
-    private $guesserLoaded = false;
-    private $typeContainer;
-    private $typeExtensionServices;
-    private $guesserServices;
+    private ?FormTypeGuesserChain $guesser = null;
```

Comment:
another place where an `is_initialized` function would be handy. then there wouldn't be a need for guesserLoaded property.